guides/plugins/apps/administration/add-custom-action-button.md

```diff
@@ -65,3 +65,40 @@ A sample payload may look like the following:
 
 Again you can verify the authenticity of the incoming request, like with [webhooks](../app-base-guide.md#webhooks), by checking the `shopware-shop-signature` it too contains the sha256 hmac of the request body, that is signed with the secret your app assigned the shop during the [registration](../app-base-guide.md#setup).
 
+The app could return a response with an empty body.
+In other way, the app should return a response with a valid body and the header `shopware-app-signature` contains the sha256 hmac of the whole response body signed with and the app secret.
+
+Examples response body:
+```json
+{
+  "actionType": "openNewTab",
```

Comment:
please explain each action type in short.
e.g. 
```
To open a new tab in the users browser you can use the `openNewTab` action type. You need to pass the url that should be openend as the `redirectUrl` property inside the payload.
```

and add similar descriptions to the other cases